debian/control

```diff
@@ -21,12 +21,12 @@ Build-Depends: debhelper (>= 10),
                libfuse3-dev,
                libprotobuf-c-dev,
                libjson-c-dev,
-               dpdk-dev,
+               dpdk-dev (<<21.11),
```

Comment:
Ahh.  Interesting that you spotted this.  DAOS doesn't have a direct dependency on DPDK right?  That's what the `daos.spec` is expressing at least.  This should likely just be removed from this file:

```suggestion
```

Reply:
done

Reply:
this change is causing build failures, reverting